test/cql-pytest/cassandra_tests/validation/entities/secondary_index_test.py

```diff
@@ -1008,8 +1007,12 @@ def testIndexOnNonFrozenCollectionOfFrozenUDT(cql, test_keyspace):
             assert_rows(execute(cql, table, "SELECT * FROM %s WHERE v CONTAINS ?", udt2), [2, {udt2}])
 
             execute(cql, table, f"DROP INDEX {test_keyspace}.{index_name}")
-            assert_invalid_message(cql, table, f"Index '{test_keyspace}.{index_name}' doesn't exist",
+
+            scylla_message = re.escape(f"Index '{index_name}' could not be found in any of the tables of keyspace '{test_keyspace}'")
+            cassandra_message = re.escape(f"Index '{test_keyspace}.{index_name}' doesn't exist")
+            assert_invalid_message_re(cql, table, '({}|{})'.format(scylla_message, cassandra_message),
```

Comment:
Nice. By the way, the new Python f-strings are even cooler than format - i.e.,
instead of 
`'({}|{})'.format(scylla_message, cassandra_message)`
You can write
`f'{scylla_message}|{cassandra_message}'`